### ISSUE TITLE
Provide methods to override the CSS in the preview

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#487]
The preview of the plugin uses the usual AsciiDoc stylesheet with minor tweaks. 

To provide a preview closer to the real result, the writer should be able to replace the CSS in the preview and/or add additional HTML header elements.

In-line with the documentation-as-code approach and the existing `.asciidoctorconfig` capabilities, this will use the existing well-known attributes `docinfo` and `stylesheet`.

**Caveats:** when the preview references an online resource like remote stylesheets of fonts, these will be fetched whenever a preview is opened.  

From the docs: 

## Custom stylesheets for the preview

The user can provide custom stylesheets for the preview to make the preview look similar to the live site they will publish the content to.

Do do this, prepare an `.asciidoctorconfig` file with settings to replace the stylesheet or to add custom headers.

WARNING: If you reference remote content like fonts or stylesheets, these will require an online connection to load them.
Therefore the user will not be able to work off-line any more.
You can provide an attribute to switch on/switch off the custom stylesheets.

```
// .asciidoctorconfigdir
// define in the AsciiDoc plugin's settings the attribute `css-online-preview`
// to show the preview using the latest stylesheets from the nightly preview.
// Will only work when online. Will pull fonts and CSS on opening the preview.
ifdef::css-online-preview[]
:stylesdir: {asciidoctorconfigdir}/.asciidoctor
:stylesheet: preview-stylesheet.css
endif::[]
```

```
/* .asciidoctor/preview-stylesheet.css */
@import url("https://mysite.com/site.css");
@import url("https://fonts.googleapis.com/css?...");
body {
  /* ... */
}
```

**Comment by ahus1** [comment]
Hi @danhaywood, 

this enhancement allows to change the CSS for the preview. As a small test-case uses the Isis site's stylesheet, see screenshot below. Changes are on a branch (no PR yet): https://github.com/apache/isis/compare/master...ahus1:custom-css-preview?expand=1

Tradeoff: the preview will look nicer when you're online. If you are offline, it will look ugly (=no CSS) as the CSS is pulled over the wire every time you open a preview. I set it up in a way that the user needs to actively switch it on by putting an attribute in the settings using ifdef. Even if the full CSS would be in a local file in the git repo, the custom fonts would still be missing.

```
ifdef::css-online-preview[]
:stylesdir: {asciidoctorconfigdir}/.asciidoctor
:stylesheet: preview-stylesheet.css
endif::[]
```

Any thoughts - is this still useful?

(the original page to compare: https://isis.apache.org/docs/2.0.0-M3/starters/helloworld.html) 

![image](https://user-images.githubusercontent.com/3957921/81335468-f1ddbb00-90a7-11ea-83dc-1c5fad429c7e.png)

**Comment by ahus1** [comment]
This is now available as part of release 0.30.71
